### ISSUE TITLE
BUG: PySide 6.5 compat

### ISSUE THREAD

**Issue by larsoner** [pyqtgraph/pyqtgraph#2674]
Using latest pyqtgraph release on all 3 major platforms see https://github.com/mne-tools/mne-qt-browser/actions/runs/4610571708/jobs/8149200467?pr=162 : 

```
mne_qt_browser/_pg_figure.py:2879: in __init__
    self.mne.plt = PlotItem(viewBox=self.mne.viewbox,
/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/pyqtgraph/graphicsItems/PlotItem/PlotItem.py:154: in __init__
    self.titleLabel = LabelItem('', size='11pt', parent=self)
/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/pyqtgraph/graphicsItems/LabelItem.py:19: in __init__
    GraphicsWidget.__init__(self, parent)
/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/pyqtgraph/graphicsItems/GraphicsWidget.py:18: in __init__
    QtWidgets.QGraphicsWidget.__init__(self, *args, **kwargs)
E   TypeError: GraphicsWidgetAnchor.__init__() takes 1 positional argument but 2 were given
```
Not sure if this is a PyQtGraph or PySide6 bug. Let me know if I should report this upstream at https://bugreports.qt.io/projects/PYSIDE/issues/

**Comment by j9ac9k** [comment]
hi @larsoner thanks for reporting.  We are having a few issues, including a segfault before the test suite so we clearly have some work to do.

**Comment by j9ac9k** [comment]
Oh wait, we may have already fixed this issue with #2662

**Comment by larsoner** [comment]
Awesome, thanks!
